linux_os/guide/services/base/service_haldaemon_disabled/rule.yml

```diff
@@ -1,6 +1,6 @@
 documentation_complete: true
 
-prodtype: rhel6
+prodtype: rhel6,rhel7,rhel8
```

Comment:
@shawndwells the haldaemon is not applicable to rhel7 or rhel8.

Reply:
Thanks for spot check. Removed from prodtype and removed CCEs.